instagram-page-react/src/pages/profilePage/organisms/posts/actions/setPosts.action.js

```diff
@@ -0,0 +1,10 @@
+import ACTION_TYPES from "../reducers/actionTypes";
+
+const setPost = (posts) => ({
```

Comment:
We should fetch the posts in the action creator so that any component that uses the posts reducer does not have to make the fetch call on its own
Please use `redux-thunk` for accomplishing the same